recipes/nest/meta.yaml

```
@@ -0,0 +1,96 @@
+{% set name = "nest-simulator" %}
+{% set version = "2.16.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nest/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: abfeb61719dec54da9477be035bef1d9d764f4e7663f63f6a6d9211f967e0490
+
+build:
+  number: 0
+  skip: true  # [win]
+  skip: true  # [osx]
+
+requirements:
+  build:
+    - cmake >=3.6
+    - git
+    - make
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - docutils
+    - future
+    - gsl
+    - icu
+    - jsoncpp
+    - libtool
+    - matplotlib
+    - mpi4py
+    - mkl
+    - ncurses
+    - numpy
+    - numpydoc
+    - openblas
+    - openmpi
+    - pandas
+    - path.py
+    - python =3.6
+    - pyqt
+    - readline
+    - scipy
+    - setuptools
+    - statsmodels
+    - zlib
+  run:
+    - cython
+    - docutils
+    - future
+    - gsl
+    - icu
+    - jsoncpp
+    - libtool
+    - matplotlib
+    - mkl
+    - mpi4py
+    - ncurses
+    - numpy
```

If this builds against `numpy` see [here](https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=blas#building-against-numpy). Else `numpy` probably isn't needed in the `host` section.
```suggestion
    - {{ pin_compatible('numpy') }}
```